src/boringssl/gen_build_yaml.py

```diff
@@ -138,6 +142,7 @@ def WriteFiles(self, files, asm_outputs):
             'flaky': False,
             'language': 'c++',
             'boringssl': True,
+            'defaults': 'boringssl',
```

Comment:
This line is probably not necessary... these are instructions for run_tests to be able to run some test binary, and not on how to build the binary.